components/explore/ExploreSort.vue

```diff
@@ -69,10 +69,6 @@ const options = computed(() => {
 function selectiveSort(options: string[]) {
   const uniqueOptions = {}
 
-  if (!isArray(options)) {
```

Comment:
Wont you it break something?

Reply:
idk.. this is a weird check to me since we expect an array as argument

Reply:
Thanks for the infos, I'm really not a fan of empty return statement but I will put it back